index.md

```diff
@@ -22,6 +22,7 @@
     RandomVectorNodeMK2
     SvBricksNode
     ImageNode
+    TorusKnotNode
```

Comment:
Sure. I can add that. What about the other nodes that don’t have Sv? (sphere, cylinder, plane etc) ?